### ISSUE TITLE
Bug re-introduced with merge in pull request.

### ISSUE THREAD

**Issue by russelljjarvis** [scidash/neuronunit#62]
This line:
https://github.com/scidash/neuronunit/blob/master/neuronunit/tests/__init__.py#L180
Notice that in my version below ```y``` is cast to quantities `pq.ms` as well as `x-offset`.
Should read. 
```
popt, pcov = curve_fit(func, x-offset*pq.ms, y*pq.ms, [0.001,2,y.min()]) # Estimate starting values for better convergence
```
I corrected this a while ago but I merged this error back in by accident after auto-merging from the merge that resulted from the pull request.

**Comment by rgerkin** [comment]
@russelljjarvis
Are you just noting that you corrected it, or do you want me to correct it?

**Comment by russelljjarvis** [comment]
Both. I corrected it my branch and remote, but it may still exist in scidash.

**Comment by rgerkin** [comment]
@russelljjarvis 
i.e. it is in the russell_dev branch but not the dev branch at scidash?

**Comment by russelljjarvis** [comment]
Oh I see you are trying to get a reference to where it is fixed in code, where as I have only given reference to where it is broken.
To be clear its broken on scidash /dev, and master:
https://github.com/scidash/neuronunit/blob/dev/neuronunit/tests/__init__.py#L215
https://github.com/scidash/neuronunit/blob/master/neuronunit/tests/__init__.py#L180

Its only reverted to my old fix in the following remote as of presently:

https://github.com/russelljjarvis/neuronunit/blob/dev/neuronunit/tests/__init__.py#L215

**Comment by rgerkin** [comment]
@russelljjarvis I looked more closely at this and there are some other bugs.  For example, it makes no sense that `y` is being multiplied by `pq.ms` because `y` is a voltage.  

So I've developed a recipe for making sure that the time constant is calculated correctly, shown [here](https://github.com/scidash/neuronunit/blob/dev/unit_test/time_constant.ipynb).  If you replace the methods in `TestPulseTest` with the corresponding methods in `Dummy` then it should work.  Try it out and see if it gives sensible results for your application.  

If it does, then I can go back and eliminate `Dummy` from that notebook and just instantiate `TimeConstantTest` instead, and the notebook will serve as a unit test for the functionality of `TimeConstantTest`.
